pennylane/devices/default_qubit_torch.py

```diff
@@ -0,0 +1,350 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This module contains a PyTorch implementation of the :class:`~.DefaultQubit`
+reference plugin.
+"""
+
+try:
+    import torch
+
+    v = torch.__version__.split(".")
+    if int(v[1]) < 8 or (int(v[1]) == 8 and int(v[2][0]) == 0):
```

Comment:
Resolved.

Reply:
I believe 1.8 and above is a minimum requirement, since this is the first Torch version to support autodifferentiation through complex matrix multiplication :)

In fact, complex matrix multiplication was only added recently in Torch 1.7, it just didn't have the differentiation capability yet!

Reply:
I see you use `semantic_version` elsewhere, but using it here will really improve readability and decrease potential mistakes.

Reply:
@albi3ro Resolved, using semantic_version now.